bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -79,6 +81,44 @@ trait UtilBijections {
   }
 
   /**
+   * Injection from twitter futures to java futures.
```

Comment:
can we add some comment that the invert here fails on any java future that is not done? This is a bit surprising to have both an `Injection` and `Bijection` between two types. I don't think this weak one should be implicit, if you ask me. It can be here without the `implicit`, but it is a bit dangerous to use.